services/API-service/src/api/event/dto/event-pcode.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsNotEmpty, IsNumber } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class EventPcodeDto {
```

Comment:
`EventPlaceCodeDto` instead of `EventPcodeDto`
`eventPlaceCodeId` instead of `eventPcodeId`